ctapipe/tools/extract_charge_resolution.py

```diff
@@ -19,36 +20,35 @@
 
 class ChargeResolutionGenerator(Tool):
     name = "ChargeResolutionGenerator"
```

Comment:
**Note OT**: This comment is not specific to this PR, I just realized it while reading this diff.

this code:

    class MyTool(Tool):
        name = "MyTool"

Seems a bit repetitive to me.  Also this line might spark the following questions in a readers mind:

 * does every `Tool` need to define a `name` class variable?
 * if so: does this `name` have to be identical to the class name or can it be chosen freely?
 * why would one use `name` and rather than `self.__class__.__name__` are there certain caveats here, where the class name is somehow altered, so that it is safer to use `self.name`?

In a word a reader might mentally trip over this line and spend some minutes to figure out the meaning of this line in the ctapipe ecosystem. And I believe it would be a nice gesture to spare the reader this time if possible.

Reply:
I thought the tool's command-line name is defined in setup.py?